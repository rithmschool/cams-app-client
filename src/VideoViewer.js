import React from "react";
import PropTypes from "prop-types";

const VideoViewer = ({ src }) => (
  <div className="lg">
    <video src={src} id="videoElement" autoPlay="true" />
    <p>
      Please take this time to look at the video viewer to make sure it is
      positioned on your face.
    </p>
    <p> Please adjust lighting so your face is clearly visible.</p>
    <p>Press the spacebar when you are ready to go forward</p>
  </div>
);

VideoViewer.propTypes = {
  src: PropTypes.string.isRequired
};

export default VideoViewer;
