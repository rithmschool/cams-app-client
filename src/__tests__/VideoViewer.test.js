import VideoViewer from "../VideoViewer";
import React from "react";
import ReactDOM from "react-dom";
import { shallow, mount } from "enzyme";
import renderer from "react-test-renderer";
import toJson from "enzyme-to-json";

test("snapshot of VideoViewer", () => {
  const videoViewer = renderer
    .create(<VideoViewer src={"something"} />)
    .toJSON();
  expect(videoViewer).toMatchSnapshot();
});
