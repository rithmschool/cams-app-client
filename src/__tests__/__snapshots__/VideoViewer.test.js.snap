// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of VideoViewer 1`] = `
<div
  className="lg"
>
  <video
    autoPlay="true"
    id="videoElement"
    muted="true"
    src="something"
  />
  <p>
    Please take this time to look at the video viewer to make sure it is positioned on your face.
  </p>
  <p>
     Please adjust lighting so your face is clearly visible.
  </p>
  <p>
    Press the spacebar when you are ready to go forward
  </p>
</div>
`;
